tests/e2e/core-tests/specs/api/order.test.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+ const { HTTPClientFactory, Order } = require( '@woocommerce/api' );
+
+ /**
+  * External dependencies
+  */
+ const config = require( 'config' );
+ const {
+	 it,
+	 describe,
+	 beforeAll,
+ } = require( '@jest/globals' );
+
+ /**
+ * Creates an order and tests interactions with it via the API.
+ */
+const runOrderApiTest = () => {
```

Comment:
OK, here we have the test for the order API! 👍 

Given that the fixture for the order (the one in `tests/e2e/config/default.json`) is pretty bare-bones (no address, no line items, therefore no taxes, etc.), I wonder whether we need to add tests for more complicated orders. Probably yes? Not in this PR, but as a follow-up.

Reply:
Follow up sounds good.

Reply:
Created an issue for this here: https://github.com/woocommerce/woocommerce/issues/29900